bin/ftscp.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/python
 
+from __future__ import print_function
```

Comment:
You can certainly do this. I will just say that WMCore did not do this fixer since it's touching a lot of our code. Hopefully CRAB3 is better